app/assets/legacy/v2/js/shared.js

```diff
@@ -440,7 +440,7 @@ window.addEventListener('load', function() {
         if (typeof web3 =='undefined'){
             $("#upper_left").addClass('disabled');
             $("#sidebar_head").html("Web3 disabled <br> <img src='/static/v2/images/icons/question.png'>");
-            $("#sidebar_p").html("Please install <a target=\"_blank\" rel=\"noopener noreferrer\" href=\"https://metamask.io/?utm_source=gitcoin.co&utm_medium=referral\">Metamask</a> <br> <a target=new href='https://github.com/gitcoinco/gitcoinco/issues/4'>What is Metamask and why do I need it?</a>.");
+            $("#sidebar_p").html("Please install <a target=\"_blank\" rel=\"noopener noreferrer\" href=\"https://metamask.io/?utm_source=gitcoin.co&utm_medium=referral\">Metamask</a> <br> <a target=new href='/web3'>What is Metamask and why do I need it?</a>.");
```

Comment:
Can we modify this to use url reverse lookup?

Reply:
its in the JS layer... not sure how we'd do that there.

Reply:
Whoops